lib/LaTeXML/Util/Unicode.pm

```diff
@@ -326,6 +324,238 @@ sub unicode_mathvariant {
   #  return $variant if $variant = $mathvariants{$font};
   return 'normal'; }
 
+#======================================================================
+our %math_props = (
+  #======================================================================
+  "0" => { role => 'NUMBER', meaning => 0 },
+  "1" => { role => 'NUMBER', meaning => 1 },
+  "2" => { role => 'NUMBER', meaning => 2 },
+  "3" => { role => 'NUMBER', meaning => 3 },
+  "4" => { role => 'NUMBER', meaning => 4 },
+  "5" => { role => 'NUMBER', meaning => 5 },
+  "6" => { role => 'NUMBER', meaning => 6 },
+  "7" => { role => 'NUMBER', meaning => 7 },
+  "8" => { role => 'NUMBER', meaning => 8 },
+  "9" => { role => 'NUMBER', meaning => 9 },
+  #======================================================================
+  '=' => { role => 'RELOP',   meaning => 'equals' },
+  '+' => { role => 'ADDOP',   meaning => 'plus' },
+  '-' => { role => 'ADDOP',   meaning => 'minus' },
+  '*' => { role => 'MULOP',   meaning => 'times' },
+  '/' => { role => 'MULOP',   meaning => 'divide' },
+  '!' => { role => 'POSTFIX', meaning => 'factorial' },
+  ',' => { role => 'PUNCT' },
+  '.' => { role => 'PERIOD' },
+  ';' => { role => 'PUNCT' },
+  ':' => { role => 'METARELOP', name     => 'colon' },          # plausible default?
+  '|' => { role => 'VERTBAR',   stretchy => 'false' },
+  '<' => { role => 'RELOP',     meaning  => 'less-than' },
+  '>' => { role => 'RELOP',     meaning  => 'greater-than' },
+  '(' => { role => 'OPEN',      stretchy => 'false' },
+  ')' => { role => 'CLOSE',     stretchy => 'false' },
+  '[' => { role => 'OPEN',      stretchy => 'false' },
+  ']' => { role => 'CLOSE',     stretchy => 'false' },
+  '{' => { role => 'OPEN',      stretchy => 'false' },
+  '}' => { role => 'CLOSE',     stretchy => 'false' },
+
+##  ':'        => { role => 'METARELOP' },    # \colon # Seems like good default role
+
+  #======================================================================
+  UTF(0x5C) => { role => 'ADDOP', meaning => 'set-minus' },        # \backslash
+  UTF(0xAC) => { role => 'BIGOP', meaning => 'not' },              # \lnot
+  UTF(0xAC) => { role => 'BIGOP', meaning => 'not' },              # \neg
```

Comment:
0xAC is given 3 separate times here? Unsure how that happened. There may be other cases, I see `0x5C` is also there twice.